plugins/dell-dock/README.md

```diff
@@ -148,3 +148,42 @@ This plugin requires read/write access to `/dev/bus/usb`.
 ## Version Considerations
 
 This plugin has been available since fwupd version `1.2.0`.
+
+## Data flow
+
+```mermaid
+  flowchart RL
+      subgraph Dell Dock
+        I2C([I2C Bus])
+        EC
+        EC_SPI[(EC\nStorage)]
+        USB1
+        USB1_SPI[(SPI)]
+        USB2
+        MST_SPI[(SPI)]
+        USB2_SPI[(SPI)]
```

Comment:
if you do the `~~~` trick to make other lines longer you can avoid the dotted lines "crossing over". Also, I used dotted lines in the SECURITY.md to mean "optional" -- maybe I need a key for that...

Reply:
Also, Mermaid is sensitive to order, so switching around the "links" can make things a lot better. Trial and error.

Reply:
Adjusted